trackerdash/README.md

```diff
@@ -0,0 +1,47 @@
+## TrackerDash JQuery Plugin
```

Comment:
Actually, the files in this directory, besides the README, were slated for deletion. I forgot to get rid of them. I'll also transfer the README to the appropriate directory, and change it to reflect this PR.